docs/versions/netstandard1.0.md

```diff
@@ -0,0 +1,15 @@
+# .NET Standard 1.0
+
+* [APIs](netstandard1.0_ref.md)
```

Comment:
Commenting here since GH won't let me comment on this file:

You could do \`\`\`csharp instead of \`\`\`diff in the _ref files to get nice syntax highlighting for C#.